src/libraries/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -500,9 +509,11 @@ public sealed partial class Interaction
         public static void AppActivate(string Title) { }
         [System.Runtime.Versioning.SupportedOSPlatformAttribute("windows")]
         public static void Beep() { }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("The procedure name cannot be statically analyzed and may be trimmed.")]
```

Comment:
It's not the procedure name that can't be statically analyzed, right? It is the type of `ObjectRef`.